proto/context/src/main/proto/spine/validation/compiled_message.proto

```diff
@@ -32,17 +32,17 @@ import "spine/options.proto";
 
 option (type_url_prefix) = "type.spine.io";
 option java_package = "io.spine.validation";
-option java_outer_classname = "MessageValidationProto";
+option java_outer_classname = "CompiledMessageProto";
 option java_multiple_files = true;
 
 import "spine/protodata/ast.proto";
 import "spine/validation/rule.proto";
 
-// All the validation rules of a single message type.
-message MessageValidation {
+// A message compiled by the validation library with its constraints, if any.
+message CompiledMessage {
```

Comment:
As long as in Spine there is a "Model Compiler" (which in turn heavily uses the Validation codegen capabilities), we cannot use `CompiledMessage`, because it clashes with that domain. Until I saw the before-after in this changeset, I had no idea what a `CompiledMessage` was. My only guess was that it was something describing the work of the Model Compiler routines completed for some particular `Message` instance, more of an item in some audit log. 

I agree that `MessageValidation` isn't the best name. But still, what this declaration represents is by no means a result of a compilation process in that sense which we assume in Model Compiler. That's why it cannot be "compiled" (a participle, meaning "it's done"), It's rather an intermediate view of a "to-do" list for the next stages.

Reply:
Let's think of a better name. I will, too.

Reply:
I thought about `CompiledMessage` as a message that is being compiled **right now**. It can also be `CompilingMessage`, `CompilationMessage`.